cassandra/src/cassandra/core.clj

```diff
@@ -198,15 +173,15 @@
 
 (defn wipe!
   "Shuts down Cassandra and wipes data."
-  [node]
+  [test node]
   (stop! node)
   (info node "deleting data files")
   (c/su
-   (meh (c/exec :rm :-r "/root/cassandra/logs"))
-   (meh (c/exec :rm :-r "/root/cassandra/data/data"))
-   (meh (c/exec :rm :-r "/root/cassandra/data/hints"))
-   (meh (c/exec :rm :-r "/root/cassandra/data/commitlog"))
-   (meh (c/exec :rm :-r "/root/cassandra/data/saved_caches"))))
+   (meh (c/exec :rm :-r (str (:cassandra-dir test) "/logs")))
+   (meh (c/exec :rm :-r (str (:cassandra-dir test) "/data/data")))
```

Comment:
Doesn't it support package-installed Cassandra ? (by something like yum or apt)
If it is supported, the directory structure should be adjusted since data directory is `/var/lib/cassandra/data`. (so no `.../data/data`)

If it's using JMX, I think it is probably safer and more robust to get the directories from JMX so that we can also remove `cassandra-dir` option.

Reply:
It doesn't support package-install for now.
I will support that for the future task.
Basically, `wipe!` is called before/after C* running.

I just modified to remove each directory in `${cassandra-dir}/data`.